tests/nat/a2a/client/conftest.py

```
@@ -0,0 +1,137 @@
+        yield {"type": "message", "content": "Streaming response"}
+
+    # Assign the actual async functions, not AsyncMock
+    mock_client._client = AsyncMock()
+    mock_client._client.send_message = mock_send_message
+    mock_client._client.send_message_streaming = mock_streaming
+
+    return mock_client
+
+
+@pytest.fixture(name="a2a_function_group")
+async def fixture_a2a_function_group(mock_a2a_client, sample_agent_card):
```

Ruff (0.14.7)

Unused function argument: `mock_a2a_client`

(ARG001)